07_memory/kernel/log.txt

```diff
@@ -0,0 +1,173 @@
+[10658.836697] 'task_05' module initialized
+[10658.836831] Test KMALLOC, number of runs: 10
+[10658.836929] 	size	  alloc, max	  alloc, min	  alloc, avg	   free, max	   free, min	   free, avg
+[10658.837145] 	2^00	17088.000 us	  352.3204550792 us	17088.000 us	 6320.000 us	  175.3204550792 us	 6320.000 us
+[10658.837425] 	2^01	 1696.000 us	  458.3204550792 us	 1696.000 us	 1072.000 us	  264.3204550792 us	 1072.000 us
+[10658.837593] 	2^02	 1040.000 us	  553.3204550792 us	 1040.000 us	  800.000 us	  346.3204550792 us	  800.000 us
+[10658.837751] 	2^03	  976.000 us	  647.3204550792 us	  976.000 us	  768.000 us	  428.3204550792 us	  768.000 us
+[10658.837969] 	2^04	 1136.000 us	  743.3204550792 us	 1136.000 us	  768.000 us	  511.3204550792 us	  768.000 us
+[10658.838172] 	2^05	 1040.000 us	  839.3204550792 us	 1040.000 us	  816.000 us	  593.3204550792 us	  816.000 us
+[10658.838384] 	2^06	 1008.000 us	  933.3204550792 us	 1008.000 us	  784.000 us	  676.3204550792 us	  784.000 us
+[10658.838585] 	2^07	 1712.000 us	 1039.3204550792 us	 1712.000 us	  832.000 us	  759.3204550792 us	  832.000 us
+[10658.838772] 	2^08	 1872.000 us	 1147.3204550792 us	 1872.000 us	  992.000 us	  845.3204550792 us	  992.000 us
+[10658.838965] 	2^09	 1616.000 us	 1250.3204550792 us	 1616.000 us	  928.000 us	  929.3204550792 us	  928.000 us
+[10658.839170] 	2^10	11424.000 us	 1510.3204550792 us	11424.000 us	  816.000 us	 1012.3204550792 us	  816.000 us
+[10658.839358] 	2^11	 1680.000 us	 1614.3204550792 us	 1680.000 us	 1248.000 us	 1103.3204550792 us	 1248.000 us
+[10658.839547] 	2^12	 1904.000 us	 1727.3204550792 us	 1904.000 us	 1152.000 us	 1190.3204550792 us	 1152.000 us
+[10658.839722] 	2^13	 1360.000 us	 1828.3204550792 us	 1360.000 us	  896.000 us	 1274.3204550792 us	  896.000 us
+[10658.839925] 	2^14	 6112.000 us	 2104.3204550792 us	 6112.000 us	 4096.000 us	 1502.3204550792 us	 4096.000 us
+[10658.840121] 	2^15	 3104.000 us	 2340.3204550792 us	 3104.000 us	 1920.000 us	 1696.3204550792 us	 1920.000 us
+[10658.840321] 	2^16	 3392.000 us	 2596.3204550792 us	 3392.000 us	 3264.000 us	 1925.3204550792 us	 3264.000 us
+[10658.840528] 	2^17	 2848.000 us	 2872.3204550792 us	 2848.000 us	 3360.000 us	 2198.3204550792 us	 3360.000 us
+[10658.840751] 	2^18	 3744.000 us	 3213.3204550792 us	 3744.000 us	 3264.000 us	 2528.3204550792 us	 3264.000 us
+[10658.840999] 	2^19	 4560.000 us	 3682.3204550792 us	 4560.000 us	 4192.000 us	 2994.3204550792 us	 4192.000 us
+[10658.841285] 	2^20	 7616.000 us	 4452.3204550792 us	 7616.000 us	 6416.000 us	 3757.3204550792 us	 6416.000 us
+[10658.841640] 	2^21	12288.000 us	 5715.3204550792 us	12288.000 us	10272.000 us	 5054.3204550792 us	10272.000 us
+[10658.842148] 	2^22	24880.000 us	 8060.3204550792 us	24880.000 us	17680.000 us	 7447.3204550792 us	17680.000 us
+[10658.842967] 	2^23	46592.000 us	12568.3204550792 us	46592.000 us	33664.000 us	12021.3204550792 us	33664.000 us
+[10658.843146] ------------[ cut here ]------------
```

Comment:
Needs rework!
In your log:
1. average always equal to max.
2. min sometimes greater than max
This is mathematically not possible.